spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/CommonBinderProperties.java

```diff
@@ -22,7 +22,7 @@
  *
  * @author Gary Russell
  */
-public interface BinderProperties {
+public interface CommonBinderProperties {
```

Comment:
+1 for both the comments. I will make this interface to `abstract` class (as used in SI) and rename the class name.